### DISCUSSION TITLE
希望ai助手可以支持腾讯混元ai和通义千问

### DISCUSSION THREAD

**Discussion by 52lemon6** [halo-dev/halo#5935] [General]
### 你当前使用的版本

2.5.2pro

### 描述一下此特性

希望ai助手可以支持腾讯混元ai和通义千问

**Comment by guqing** [comment]
ping @LIlGG

**Comment by LIlGG** [comment]
可以考虑添加，但目前 AI 相关产品较多，可能不会全部进行适配。

**Comment by lhDream** [comment]
@52lemon6 
可以通过OpenAI 兼容方式接入,, 目前主流的AI产品应该都支持openapi
